src/api/treeTrackerApi.js

```diff
@@ -18,7 +18,26 @@ export default {
   makeQueryString(filterObj) {
     let arr = [];
     for (const key in filterObj) {
-      if ((filterObj[key] || filterObj[key] === 0) && filterObj[key] !== '') {
+      if (key === 'device_identifier') {
```

Comment:
Use [`encodeURIComponent()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) instead.
Assuming the API supports it, it would be a good idea to do this with all `string` filters.

Reply:
Oh, nice! Thanks